filechooser.py

```diff
@@ -47,7 +47,11 @@ def update_preview_cb(file_chooser, preview):
         try:
             i = Image.open(filename)
             r = cropgui_common.image_rotation(i)
-            i.thumbnail((PREVIEW_SIZE, PREVIEW_SIZE), Image.Resampling.LANCZOS)
+            if hasattr(Image, 'Resampling'):
+                thumb_filter = Image.Resampling.LANCZOS
+            else:
+                thumb_filter = Image.LANCZOS
+            i.thumbnail((PREVIEW_SIZE, PREVIEW_SIZE), thumb_filter)
```

Comment:
```suggestion
            try:
                thumb_filter = Image.Resampling.LANCZOS
            except AttributeError:
                thumb_filter = Image.LANCZOS
            i.thumbnail((PREVIEW_SIZE, PREVIEW_SIZE), thumb_filter)
```
Or something more pythonic?

Reply:
I'm no Python expert, but after some googling I found another potential solution:

```python
getattr(Image, 'Resampling', Image).LANCZOS
```

This is shorter and also more performant. Lmk what you prefer.